Code/src/main/java/nl/utwente/group10/ui/gestures/CustomGesture.java

```diff
@@ -12,16 +12,16 @@
 import javafx.scene.input.MouseEvent;
 import javafx.scene.input.TouchEvent;
 
-public class CustomGesture implements EventHandler<MouseEvent> {
-
-	private GestureCallBack callBack;
+public class CustomGesture extends AbstractGesture implements EventHandler<MouseEvent> {
 	private long startTime;
 	private Date date = new Date();
 	private boolean draggedWhilePressed;
+	private GestureCallBack callBack;
 
 	public CustomGesture(GestureCallBack callBack, Node latchTo) {
+		super(latchTo);
 		this.callBack = callBack;
-		latchTo.addEventHandler(MouseEvent.ANY, this);
+		System.out.println("Created!");
```

Comment:
Dit is een klasse die er nog van Kristel in staat. Wordt momenteel niet meer gebruikt